acid/db.py

```diff
@@ -13,12 +13,16 @@
 
 def connect():
     try:
-        db.bind(provider=config['database']['provider'],
-                host=os.getenv('DBHOST', config['database']['host']),
-                port=os.getenv('DBPORT', config['database']['port']),
-                user=os.getenv('DBUSER', config['database']['user']),
-                passwd=os.getenv('DBPASSWORD', config['database']['password']),
-                db=os.getenv('DBNAME', config['database']['name']))
+        db.bind(provider=current_app.config['database']['provider'],
```

Comment:
You could add 1 line instead of changing 6 other lines.

All you needed was to add:
```python
config = current_app.config
```